src/bed.cpp

```diff
@@ -67,18 +69,33 @@ Attr_ReadValue BedItem::readAttr(AttrTypes_t attr, PropStream& propStream)
 	return Item::readAttr(attr, propStream);
 }
 
-void BedItem::serializeAttr(PropWriteStream& propWriteStream) const
+bool BedItem::unserializeAttr(const tfs::Item& pbItem)
```

Comment:
I think it would be nice if the serialization code were placed in a subnamespace of tfs instead - I plan to wrap our code into the tfs namespace, so I wouldn't want any collisions.